app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -48,7 +47,7 @@ class MainActivity : ComponentActivity() {
 }
 
 @Composable
-fun Header(name: String) {
+fun Header() {
```

Comment:
Rewrote this class header to get rid of the name: String as it was considered redundant by the IDE

Reply:
Yeah... looks like it's not being used.  Might as well remove it.  Less code is usually better.